catalogs/catalog-lakehouse-iceberg/src/main/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/ops/IcebergTableOps.java

```diff
@@ -140,6 +148,59 @@ public void close() throws Exception {
     if (catalog instanceof AutoCloseable) {
       // JdbcCatalog need close.
       ((AutoCloseable) catalog).close();
+
+      // Because each catalog in Gravitino has its own classloader, after a catalog is longer used
+      // for a long time or dropped, the instance of classloader needs to be released. In order to
+      // let JVM GC remove the classloader, we need to release the resources of the classloader. The
+      // resources include the driver of the catalog backend and the
+      // AbandonedConnectionCleanupThread of MySQL. For me information about
+      // AbandonedConnectionCleanupThread, please refer to the corresponding java doc of MySQL
+      // driver.
+      if (backendUri != null) {
```

Comment:
I prefer to check catalog type explictly not to deponds on `catalog instanceof AutoCloseable`

Reply:
OK